mypy/build.py

```diff
@@ -1353,6 +1367,18 @@ def strongly_connected_components_path(vertices: Set[str],
                                        edges: Dict[str, List[str]]) -> Iterator[Set[str]]:
     """Compute Strongly Connected Components of a graph.
 
+    The graph is a DAG.
```

Comment:
What does this mean? The resulting graph where nodes are strongly connected components is a graph?